autoload/SpaceVim/layers/lang/csharp.vim

```diff
@@ -0,0 +1,113 @@
+"=============================================================================
+" csharp.vim --- SpaceVim lang#csharp layer
+" Copyright (c) 2016-2018 VyronLee & Contributors
+" Author: VyronLee < lwz_jz@hotmail.com >
+" URL: https://spacevim.org
+" License: GPLv3
+"=============================================================================
+
+""
+" @section lang#csharp, layer-lang-csharp
+" @parentsection layers
+" This layer includes utilities and language-specific mappings for csharp development.
+"
+" @subsection Key Mappings
+" >
+"   Mode            Key             Function
+"   ---------------------------------------------
+"   normal          SPC l b         compile the project
+"   normal          SPC l f         format current file
+"   normal          SPC l d         show doc
+"   normal          SPC l g g       go to definition
+"   normal          SPC l g i       find implementations
+"   normal          SPC l g t       find type
+"   normal          SPC l g s       find symbols
+"   normal          SPC l g u       find usages of symbol under cursor
+"   normal          SPC l g m       find members in the current buffer
+"   normal          SPC l r r       rename symbol under cursor
+"   normal          SPC l s r       reload the solution
+"   normal          SPC l s s       start the OmniSharp server
```

Comment:
I recommend to use same language specific key bindings as other language, then when user edit more then one languages, they do not need to remember key bindings.

Reply:
ok, I will modify it and recommit later.

Reply:
@wsdjeg How about the non suggested key bindings? Can I still use 2-level binding, such as ` SPC l s r   -> reload the solution`?